app/src/main/res/layout/product_item.xml

```diff
@@ -0,0 +1,128 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:background="@color/colorWhite"
+    android:layout_marginTop="@dimen/adapter_margin"
+    android:layout_marginEnd="@dimen/adapter_margin"
+    android:layout_marginStart="@dimen/adapter_margin"
+    android:orientation="vertical">
+
+    <EditText
+        android:id="@+id/et_quantity"
+        android:layout_width="72dp"
```

Comment:
Inside this file there are a lot of number that must be go inside a dimens.xml file, please move them